cmdline/bin/dcat.dart

```diff
@@ -36,34 +36,35 @@ void main(List<String> arguments) {
   dcat(paths, argResults[LINE_NUMBER]);
 }
 
-Future dcat(List<String> paths, bool showLineNumbers) {
+Future dcat(List<String> paths, bool showLineNumbers) async {
```

Comment:
After further discussion, I believe we've decided that async is an implementation detail that doesn't affect the method signature, so you *should* specify the return value.